### PR TITLE
Change "Ask a question" String to Translation Function Call

### LINKED ISSUE #10031: Change "Ask a question" String to Translation Function Call
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all). This means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**. 

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new**, and looking through our general [help wanted](https://github.com/publiclab/plots2/labels/help-wanted) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Comment below. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!

- [ ] 📝 **Update** the file [question.html.erb](https://github.com/publiclab/plots2/blob/2bb0cdd28c8d1a513b0161c402062f4edee4624d/app/views/editor/question.html.erb#L15)  in the `plots2` repository (press the little pen Icon) and edit the line as shown below. 

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:
Change this [line](https://github.com/publiclab/plots2/blob/2bb0cdd28c8d1a513b0161c402062f4edee4624d/app/views/editor/question.html.erb#L15)  as shown in git diff
```diff
-    <h4 class="d-lg-none">Ask a question</h4>
+    <h4 class="d-lg-none"><%=translation('dashboard._header.dropdown.ask_question')%></h4>
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there. 

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance! 

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it. 

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below!

### PR DESCRIPTION
**Pull request by Onyinye91-ctrl**
I changed  "Ask a question" String to Translation Function Call like was instructed

Fixes #10031>

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ✔️] PR is descriptively titled 📑 and links the original issue above 🔗
* [✔️ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ✔️] code is in uniquely-named feature branch and has no merge conflicts 📁
* [✔️ ] screenshots/GIFs are attached 📎 in case of UI updation
* [✔️ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

File: app/models/drupal_content_type_map.rb [hunk 1]
```diff
@@ -29,9 +29,9 @@ def cartographer_notes
   end
 
   def license
-    l = "<a href='http://creativecommons.org/publicdomain/zero/1.0/'>Public Domain</a>" if field_license_value == 'publicdomain'
-    l = "<a href='http://creativecommons.org/licenses/by/3.0/'>CC-BY</a>" if field_license_value == 'cc-by'
-    l = "<a href='http://creativecommons.org/licenses/by-sa/3.0/'>CC-BY-SA</a>" if field_license_value == 'cc-by-sa'
+    l = "<a href='https://creativecommons.org/publicdomain/zero/1.0/'>Public Domain</a>" if field_license_value == 'publicdomain'
+    l = "<a href='https://creativecommons.org/licenses/by/3.0/'>CC-BY</a>" if field_license_value == 'cc-by'
+    l = "<a href='https://creativecommons.org/licenses/by-sa/3.0/'>CC-BY-SA</a>" if field_license_value == 'cc-by-sa'
```
Comment (noi5e) [inline app/models/drupal_content_type_map.rb:34]: @Onyinye91-ctrl Thanks for changing these, even though this isn't what the original issue was asking for, I think this will still work.

File: app/models/node.rb [hunk 2]
```diff
@@ -681,7 +681,7 @@ def self.new_note(params)
                     comment: 2,
                     type:    'note')
     node.status = 4 if author.first_time_poster
-    node.draft if params[:draft] == "true"
+    node.save_draft if params[:draft] == "true"
```
Comment (noi5e) [inline app/models/node.rb:685]: @Onyinye91-ctrl, I'm not sure why these changes were made, since the original issue didn't mention them? Can you revert these changes in another commit?

File: app/views/editor/question.html.erb [hunk 3]
```diff
@@ -12,7 +12,7 @@
   <%= javascript_include_tag "dragdrop" %>
 
   <% if params[:template] == "question" %>
-    <h4 class="d-lg-none">Ask a question</h4>
+    <h4 class="d-lg-none"><%=translation('dashboard._header.dropdown.ask_question')%></h4>
```
Comment (noi5e) [inline app/views/editor/question.html.erb:15]: This looks good to me!

**Comment by noi5e** [comment]
@Onyinye91-ctrl Oh, I see! I looked through the commit history here, and it looks like the branch `Onyinye91-ctrl:main` that you're merging from includes changes that you made last year.

Screen Shot 2021-08-22 at 9 40 08 AM

Ideally, there should only be one File Changed here (`app/views/editor/question.html.erb`) before we merge it:
Screen Shot 2021-08-22 at 9 44 19 AM

I think it would be best if you open another pull request merging from a fresh branch. 

Here are some steps you can take to do that:

1. Revert all your changes to `Onyinye91-ctrl:main`. You can do that by doing `git checkout -- [filename]` for each file you've changed.
2. Run a `git fetch upstream main` then `git merge upstream/main` to fetch the most recent version of `plots2:main`.
3. Switch to a new branch with `git checkout -b translation-question-string`.
4. Make your changes to `question.html.erb` and then submit a pull request from that branch.

I know that's a lot more work, but that way we can be sure we're only making changes where we need to make them 🤗

**Comment by Tlazypanda** [comment]
Hey @Onyinye91-ctrl hope you have seen @noi5e response. If you have any questions feel free to reach out :) We are here to help :heart: Another hack to reset this although not recommended would be to delete the fork and freshly fork the repo and make the changes :v: This can allow you to make changes within github without invoking git

**Comment by noi5e** [comment]
Hi @Onyinye91-ctrl, it looks like you opened another PR for this at #10086, so I'm going to close this PR and continue the conversation over there!
